### PR TITLE
quick docs fix to remove assumption that we're using kind

### PR DESCRIPTION
**Pull request by ashleywang1**
/skip-changelog

### CODE REVIEW

**Comment by ashleywang1** [comment]
Looks like this:
Screen Shot 2021-04-13 at 4 29 58 PM

File: docs/content/guides/locality_routing.md [hunk 1]
```diff
@@ -22,22 +22,32 @@ To illustrate these concepts, we will assume that:
...
+{{< /tab >}}
+{{< /tabs >}}
 
 ## Configure the Region and Zone for the Nodes
 
 Gloo Mesh uses the configured [region and zone labels](https://v1-18.docs.kubernetes.io/docs/reference/kubernetes-api/labels-annotations-taints/#topologykubernetesioregion) on nodes to indicate locality for services. If you do not already have the region and zone labels set, you will need to do so now. In our example, we will set the `mgmt-cluster` node to use `us-east-1` for the region and `us-east-1a` for the zone. The `remote-cluster` node will be set to `us-east-2` and `us-east-2b` respectively. In a cloud-based deployment, these labels will typically be set by the cloud provider.
 
 ```bash
-kubectl label node $MGMT_CLUSTER-control-plane --context kind-$MGMT_CLUSTER \
+kubectl label node $MGMT_CLUSTER-control-plane --context $MGMT_CLUSTER_NAME \
```
Comment (EItanya) [inline docs/content/guides/locality_routing.md:47]:
```suggestion
kubectl label node $MGMT_CLUSTER-control-plane --context $MGMT_CONTEXT \
```

File: docs/content/guides/locality_routing.md [hunk 2]
```diff
@@ -22,22 +22,32 @@ To illustrate these concepts, we will assume that:
...
   topology.kubernetes.io/region=us-east-1 topology.kubernetes.io/zone=us-east-1a
 
-kubectl label node $REMOTE_CLUSTER-control-plane --context kind-$REMOTE_CLUSTER \
+kubectl label node $REMOTE_CLUSTER-control-plane --context $REMOTE_CLUSTER_NAME \
```
Comment (EItanya) [inline docs/content/guides/locality_routing.md:50]:
```suggestion
kubectl label node $REMOTE_CLUSTER-control-plane --context $REMOTE_CONTEXT \
```

**Review by jenshu — APPROVED** [review]

**Pull request merged**